pkg/kvstore/store/syncstore.go

```diff
@@ -110,15 +135,20 @@ func NewWorkqueueSyncStore(backend SyncStoreBackend, prefix string, opts ...WSSO
 		opt(wss)
 	}
 
+	wss.log = wss.log.WithField(logfields.ClusterName, wss.source)
 	wss.workqueue = workqueue.NewRateLimitingQueue(wss.limiter)
 	wss.queuedMetric = metrics.KVStoreSyncQueueSize.WithLabelValues(kvstore.GetScopeFromKey(prefix), wss.source)
+	wss.syncedMetric = metrics.KVStoreInitialSyncCompleted.WithLabelValues(kvstore.GetScopeFromKey(prefix), wss.source, "write")
```

Comment:
Do we need the `action` label if the only value is ever `write`?

Reply:
This PR deals with the "write" side, while https://github.com/cilium/cilium/pull/25499 will introduce the "read" side.

Reply:
Do we envision `action` ever being more than just `read` or `write`?

Reply:
Personally I don't think so.

Reply:
I'm trying to decide if they should be two separate metrics or one. Either works, I'm currently leaning towards one metric.